src/utils/FilterBy.tsx

```diff
@@ -0,0 +1,5 @@
+export enum FilterBy {
```

Comment:
Consider renaming the enum 'FilterBy' to better reflect its usage, such as 'TodoFilter'.